graphql/tests/taskLogs/results.json

```diff
@@ -47,7 +47,7 @@
                 "data": {
                   "hostId": "hostid",
                   "jiraIssue": "an_issue",
-                  "jiraLink": "https:///browse/an_issue",
+                  "jiraLink": "https://https://jira.mongodb.org/browse/an_issue",
```

Comment:
What's the reason for needing our jira instance in test data?

Reply:
it's automatically put there by testutil